config/gen2convert.yaml

```diff
@@ -37,18 +37,21 @@ collections:
     -
       pattern: "^(.+)/rerun/"
       repl: "shared/"
+    -
+      pattern: "^(.+)/CALIB"
+      repl : "calib"
```

Comment:
I'm running tests to see if this is the problem I think it is, but in DM-16467, I've had to change the collection of calibration data to match the input data to get things to work.  I don't know if this is still the case now (hence the testing), but has there been a discussion of this: datasets that should be members of all collections?

Reply:
The vision was instead to permit multiple input collections, which would be treated as a priority-ordered search path.  It looks like that's doable with `--input collection1,collection2`, though I'm not completely certain it's been implemented.  Have you tried that?  Or is there a usability problem with that approach that you've identified?

Reply:
I haven't tried using the comma separated version, but if that's the plan, then I'm happy to use that.  I need to rebuild my test environment to confirm this will work with the ISR pipelineTask.